### PR TITLE
Rename mint- prefix to sg- in helpers and docs

### PR DESCRIPTION
**Pull request by matzimowski**

### CODE REVIEW

File: src/components/buttons/icon-as-button.html [hunk 1]
```diff
@@ -203,6 +203,5 @@ <h3 class="docs-block__header">xsmall</h3>
         </svg>
       </div>
     </button>
-    </div>
```
Comment (aju) [inline src/components/buttons/icon-as-button.html]: waaat ?

File: src/docs/_sass/_color-box.scss [hunk 2]
```diff
@@ -51,7 +51,7 @@
     content: '$mintPrimary';
   }
 }
-.color-box--mint-mid-tone {
+.color-box--sg-mid-tone {
```
Comment (vergilius) [inline src/docs/_sass/_color-box.scss]: this is wrong, should be mint ❗

File: src/docs/_sass/_color-box.scss [hunk 3]
```diff
@@ -60,7 +60,7 @@
     content: '$mintSecondary';
   }
 }
-.color-box--mint-light {
+.color-box--sg-light {
```
Comment (vergilius) [inline src/docs/_sass/_color-box.scss]: also here

**Comment by aju** [comment]
ok

**Comment by kdzwinel** [comment]
lovely 👍

File: src/docs/_sass/_color-box.scss (hunk 2 above)
Comment (matzimowski) [inline src/docs/_sass/_color-box.scss]: 👍

File: src/components/buttons/icon-as-button.html (hunk 1 above)
Comment (matzimowski) [inline src/components/buttons/icon-as-button.html]: yep, it's not my bug. it was breaking layout...

File: src/docs/_sass/_color-box.scss (hunk 3 above)
Comment (matzimowski) [inline src/docs/_sass/_color-box.scss]: done

**Comment by vergilius** [comment]
OK

**Pull request merged**